.github/workflows/cd.yml

```diff
@@ -1,35 +1,28 @@
-# This workflow will build a Java project with Gradle
-# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
-
 name: CD
 
 on:
   workflow_dispatch:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-    env:
-      TEST_AUTH_TOKEN: ${{ secrets.ALPHA_TEST_AUTH_TOKEN }}
-      TEST_CACHE_NAME: dummy
 
     steps:
-    - uses: actions/checkout@v2
+    - name: Checkout project
+      uses: actions/checkout@v3
       with:
         token: ${{ secrets.MOMENTO_MACHINE_USER_GITHUB_TOKEN }}
-        submodules: true
-    - name: Set up JDK 16
-      uses: actions/setup-java@v2
+
+    - name: Set up JDK 17
```

Comment:
I updated to Java 17 since 16 wasn't an lts release. I could try 20 but that is only 3 days old.
I switched to corretto since AdoptOpenJdk isn't developed anymore. We can use any version supported by the setup java plugin if we have a strong preference. [https://github.com/actions/setup-java#supported-distributions](url)

Reply:
nope 17 is good